news/2025/2025-04-10-osutaiko-world-cup-quarterfinals-recap.md

```diff
@@ -0,0 +1,153 @@
+| United States ::{ flag=US }:: | 5 | **6** | ::{ flag=CL }:: **Chile** |
+| **France** ::{ flag=FR }:: | **6** | 1 | ::{ flag=NZ }:: New Zealand |
+| **Japan** ::{ flag=JP }:: | **6** | 0 | ::{ flag=MY }:: Malaysia |
+| **Türkiye** ::{ flag=TR }:: | **6** | 1 | ::{ flag=ID }:: Indonesia |
+| **Russian Federation** ::{ flag=RU }:: | **6** | 1 | ::{ flag=FI }:: Finland |
+| **United Kingdom** ::{ flag=GB }:: | **6** | 0 | ::{ flag=SK }:: Slovakia |
+
+Switching things up to a Best of 13 format, the Quarterfinals had some very close matches for some teams. Interestingly enough: every time the tiebreaker was played, the lower seeded team came out as the victor. Not all of them were able to pull through and come out on top however, so the following teams were unfortunately eliminated: ::{ flag=PH }:: Philippines, ::{ flag=CH }:: Switzerland, ::{ flag=HK }:: Hong Kong, ::{ flag=VN }:: Vietnam, ::{ flag=PL }:: Poland, ::{ flag=CA }:: Canada, ::{ flag=AU }:: Australia, ::{ flag=NL }:: Netherlands, ::{ flag=FI }:: Finland, ::{ flag=SK }:: Slovakia, ::{ flag=ID }:: Indonesia & ::{ flag=NZ }:: New Zealand.
+
+Check out detailed statistics for this round [here](https://docs.google.com/spreadsheets/d/1HYP2llOprad492cK_JaKEV9B79wMk1OYN8c8xSpa2xI).
+
+## Match highlights
+
+### ::{ flag=AU }:: Australia vs. ::{ flag=FR }:: France
+
+**[View the match information here!](https://osu.ppy.sh/community/matches/117698638)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=0h4m42&video=2424570850" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Wrapping up this train of tiebreaker upsets, we have perhaps the most surprising of the week: Team Australia (#8) vs. Team France (#19) — not only the largest difference in seeding, but with Team France holding a total of zero pick'em votes, truly nobody saw this coming.
+
+Australia began by playing it safe with [Marmalade Butcher - =/=Virtual Futural](https://osu.ppy.sh/beatmapsets/2347292#taiko/5048649) (NM1), taking their first point while testing the waters. However, France quickly showed that Australia might have gone fishing in the wrong lake, with a 100,000 point win on [Yunosuke - Ziqqurat (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2347287#taiko/5048627) (NM6).
+
+Having stirred the pot, the two teams began trading points: Australia winning [Mono. - skip&step](https://osu.ppy.sh/beatmapsets/2069452#taiko/4330009) (HR1), France taking [Hoshimachi Suisei - BIBBIDIBA(AL-1S & UZQueen's Edit)](https://osu.ppy.sh/beatmapsets/2207301#taiko/4673960) (NM2), followed by an Australian accuracy advantage on [DARK_GENE - GREEN MAMMALS](https://osu.ppy.sh/beatmapsets/2347276#taiko/5048593) (FM1), and an extremely narrow five-thousand-point French victory on [Pmarusama. - Tokimeki Broker](https://osu.ppy.sh/beatmapsets/2189313#taiko/4630501) (DT2).
```

Comment:
still in the thousands so the spelled-out emphasis only makes it slightly harder to parse the numeral at a glance

```suggestion
Having stirred the pot, the two teams began trading points: Australia winning [Mono. - skip&step](https://osu.ppy.sh/beatmapsets/2069452#taiko/4330009) (HR1), France taking [Hoshimachi Suisei - BIBBIDIBA(AL-1S & UZQueen's Edit)](https://osu.ppy.sh/beatmapsets/2207301#taiko/4673960) (NM2), followed by an Australian accuracy advantage on [DARK_GENE - GREEN MAMMALS](https://osu.ppy.sh/beatmapsets/2347276#taiko/5048593) (FM1), and an extremely narrow 5,000-point French victory on [Pmarusama. - Tokimeki Broker](https://osu.ppy.sh/beatmapsets/2189313#taiko/4630501) (DT2).
```